src/components/footer/Footer.js

```diff
@@ -1,22 +1,67 @@
 import React from 'react';
-import { translate } from 'react-i18next';
+import { translate, Interpolate } from 'react-i18next';
 import 'i18n/i18n';
+import Panel from 'components/panel/Panel';
+import Button from 'components/button/Button';
 import styles from './footer.scss';
+import constant from 'config/constants';
 
-const Footer = ({ t }) => (
-  <footer className={styles.footerMain}>
-    <div className={styles.footerInner}>
-      <a href="#terms" className={styles.footerLink}>
-        {t('footer-terms')}
+const Footer = ({ t }) => {
+  const createTrigger = (type) => {
+    return (
+      <a href={'#' + type} className={styles.footerLink}>
+        {t(`footer-${type}`)}
       </a>
-      <a href="#cookies" className={styles.footerLink}>
-        {t('footer-cookies')}
-      </a>
-      <a href="#privacy" className={styles.footerLink}>
-        {t('footer-privacy')}
-      </a>
-    </div>
-  </footer>
-);
+    );
+  };
+
+  const createPanel = (type, detail) => (
+    <Panel trigger={createTrigger(type)} modal closable closeAction={closeBtn} key={type}>
+      <Panel.Header>{t(`footer-${type}`)}</Panel.Header>
+      <Panel.Content classes={classes}>
+        <p>
+          <Interpolate
+            i18nKey={detail}
+            lineBreak={doubleLineBreak}
+            freePrivacyPolicy={freePrivacyPolicyLink}
+          />
+        </p>
+      </Panel.Content>
+    </Panel>
+  );
+
+  const doubleLineBreak = (
```

Comment:
Do we really need this? Can we do it with css only?

Reply:
I am not sure. I will be more than happy if we can 🤞

Reply:
Should we have rich text for policies (cookies, TnC etc.) instead of plain text. Its really a pain to style plain text 😞 
But I am not sure if this is correct approach (due to XSS attack) Any thoughts @crysfel @tureey @emmawedekind